packages/core/src/schema/models/evaluationTemplates.ts

```diff
@@ -9,6 +9,7 @@ export const evaluationTemplates = latitudeSchema.table(
     id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
     name: varchar('name', { length: 256 }).notNull(),
     description: text('description').notNull(),
+    category: varchar('category', { length: 256 }).notNull(),
```

Comment:
Are we sure we don't want to have some categories table to make filters easier?

Reply:
Could do, but that would add `JOIN` statements to the selects. I do like it more as foreign keys, but I'm not sure if we really do care about it, given that it affects (little) to the query performance.

Reply:
I mean unstructured category not sure if it's that helpful. Also how is category loaded? Maybe is not necessary on evaluation detail or can be loaded lazy

And lastly if in the future we want to to have multiple categories it would be easier if this is already a table

Reply:
👌